test/testcafe/framework/page-objects/FactorEnrollPasswordPageObject.js

```diff
@@ -38,6 +39,10 @@ export default class EnrollPasswordPageObject extends BasePageObject {
     return this.form.clickSaveButton();
   }
 
+  clickResetPasswordButton() {
+    this.form.clickButton(resetPasswordButtonName);
```

Comment:
nit: inline the strings here since it's only used once

Reply:
its also used below on line 80

Reply:
Ah I misread the code too quickly. then this is a 50/50 to me, either works.